bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperClientStats.java

```diff
@@ -93,6 +93,9 @@
     String FAILED_CONNECTION_COUNTER = "FAILED_CONNECTION_COUNTER";
     String FAILED_TLS_HANDSHAKE_COUNTER = "FAILED_TLS_HANDSHAKE_COUNTER";
 
+    // placementpolicy stats
+    String NUM_BOOKIES_IN_DEFAULT_FAULTZONE = "NUM_BOOKIES_IN_DEFAULT_FAULTZONE";
```

Comment:
Shouldn't we have similar stats for each fault zone, including number of writable bookies there? Then we could alert on a fault zone not having enough (writable) bookies in a fault zone.

Reply:
+1

Reply:
responded already - https://github.com/apache/bookkeeper/pull/1941#issuecomment-462951665